manifest.json

```diff
@@ -1,27 +1,28 @@
 {
-
-  "manifest_version": 2,
-  "name": "wxIF",
-  "version": "0.2",
+  "applications": {
+    "gecko": {
+      "id": "wxif@morbo.org",
+      "strict_min_version": "48.0"
+    }
+  },
+  "author": "Gian-Carlo Pascutto & Martin Baranski",
```

Comment:
Not sure if adding everybody who contributes patches is doable in the long run.

Reply:
Sound's good to me. I'll update the file ASAP. 😃

Reply:
Fixed. 😃